.github/ionic-issue-bot.yml

```diff
@@ -11,15 +11,15 @@ closeAndLock:
         for questions about Stencil.
```

Comment:
should we say 'Stencil and Stencil Store' here?

Reply:
Yep, I think so!